client/reader/sidebar/index.jsx

```diff
@@ -254,22 +235,51 @@ const ReaderSidebar = React.createClass( {
 					onTagExists={ this.highlightNewTag }
 					currentTag={ this.state.currentTag } />
 
-				</SidebarRegion>
+			</SidebarRegion>
 
-				{ this.renderAppPromo() }
+			{ this.props.shouldRenderAppPromo &&
+				<div className="reader-sidebar__AppPromo">
```

Comment:
Very soon this will become a lint warning: https://github.com/Automattic/eslint-plugin-wpcalypso/pull/27

Class names should be lower-case and dash-delimited (aside from namespace `__` prefix separator).